circle.yml

```diff
@@ -15,11 +14,14 @@ dependencies:
 test:
   override:
     - bundle exec rake
-    - docker build -t=$registry_root/$image_name:b$CIRCLE_BUILD_NUM .
+    - docker build -t=$registry_root/$CIRCLE_PROJECT_REPONAME:b$CIRCLE_BUILD_NUM .
```

Comment:
Any variables used here will have to be set in the circle.yml or it will fail on contributor's PRs (since it'll be unset for those builds). That's the reason it was moved in the CLI repo.

I'd also suggest renaming to `REGISTRY_ROOT` (and `PUBLIC_REGISTRY_ROOT`) since environment variables are conventionally capitalized.